app/src/main/java/com/example/shorebuddy/adapters/LakeListAdapter.java

```diff
@@ -36,7 +35,7 @@ private LakeViewHolder(View itemView, OnLakeListener onLakeListener) {
     private final OnLakeListener onLakeListener;
     private List<Lake> lakes;
 
-    public LakeListAdapter(Context context, OnLakeListener onLakeListener) {
+        public LakeListAdapter(Context context, OnLakeListener onLakeListener) {
```

Comment:
Is this an accidental tab?

Reply:
Most likely, I was trying to understand the implementation of the back end of with the implementation of the Views and ViewModels and that probably when this happened